iac-extensions/docker/src/main/java/org/sonar/iac/docker/parser/grammar/DockerGrammar.java

```diff
@@ -229,19 +236,25 @@ public PortTree PORT() {
 
   public LabelTree LABEL() {
     return b.<LabelTree>nonterminal(DockerLexicalGrammar.LABEL).is(
-      f.label(b.token(DockerKeyword.LABEL),
-        b.oneOrMore(
-          b.firstOf(KEY_VALUE_PAIR_WITH_EQUALS(), KEY_VALUE_PAIR())
+      f.label(
+        b.token(DockerKeyword.LABEL),
+        b.token(DockerLexicalGrammar.SPACE),
```

Comment:
I added this `SPACE` specifically because in some cases KeyValuePair should not have trailing spaces before.
I couldn't find a better solution than having a naïve regex, as using other format was failing the parsing :/